### PR TITLE
Find the 3rd largest element in the array

### PR DESCRIPTION
**Pull request by vishalv971**

### CODE REVIEW

**Comment by vishalv971** [comment]
@SaumyaSachdev  and @pranjalkaler  
Answer to example in `main()` is 2.

File: LeetCode/3rdLargestElement.cpp [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+            return
+                it->first;
+
+        }
+};
+
+int main() {
+    std::vector<int> v = {1,2,2,5,3,5};
+    Solution s; 
+    cout<<s.thirdMax(v)<<endl;
```
Comment (vishalv971) [inline LeetCode/3rdLargestElement.cpp:52]: Will print 2

Comment (pranjalkaler) [reply LeetCode/3rdLargestElement.cpp:52]: Can we generalize the code to print nth largest element?
Passing n as second argument to the function?

Comment (vishalv971) [reply LeetCode/3rdLargestElement.cpp:52]: Sounds good @pranjalkaler  , won't amend the commit  will add a new commit.

Comment (pranjalkaler) [reply LeetCode/3rdLargestElement.cpp:52]: sounds like a plan

**Review by pranjalkaler — APPROVED** [review]

**Pull request merged**